pkg/v22/chartconfig/current.go

```diff
@@ -15,6 +15,7 @@ import (
 	"github.com/giantswarm/cluster-operator/pkg/label"
 	"github.com/giantswarm/cluster-operator/pkg/project"
 	"github.com/giantswarm/cluster-operator/pkg/v22/controllercontext"
+	pkgerrors "github.com/giantswarm/cluster-operator/pkg/v22/errors"
```

Comment:
Moving here since the error handling is needed in multiple packages but only for cluster-operator.